### PR TITLE
feat(proto): Wrap string and int32 fields where necessary

### PR DESCRIPTION
**Pull request by ezimanyi**
* feat(kubernetes): Add missing Kubernetes fields

  A few fields were missing from KubernetesAccount. Add these.

* feat(proto): Wrap string and int32 fields where necessary

  This commit goes through all string and int32 fields, wrapping them where necessary. The only changes necessary were a few string fields.

### CODE REVIEW

**Comment by ezimanyi** [comment]
Just enums left after this one!

File: api/proto/cloudprovider/kubernetes.proto [hunk 1]
```diff
@@ -83,6 +83,51 @@ message KubernetesAccount {
...
+
+    // If set, all calls to kubectl will time out after the specified number of
+    // seconds.
+    int32 kubectlRequestTimeoutSeconds = 22;
+
+    // Whether to check whether the account has permission to read configured
+    // kinds before caching them. Kinds that the account does not have permission
+    // to read will be omitted from caching.
+    // This field defaults to true, and it is recommended to leave it at the
+    // default. If this field is set to false, any kubernetes objects that
```
Comment (maggieneterval) [inline api/proto/cloudprovider/kubernetes.proto]:
```suggestion
    // default. If this field is set to false, any Kubernetes objects that
```

File: api/proto/cloudprovider/kubernetes.proto [hunk 2]
```diff
@@ -83,6 +83,51 @@ message KubernetesAccount {
...
+    // service account to use.
+    string oAuthServiceAccount = 24;
+
+    // If true, only cache Kubernetes objects that have been deployed by
+    // Spinnaker, and ignore any other objects that exist in the cluster.
+    // Defaults to false.
+    google.protobuf.BoolValue onlySpinnakerManaged = 27;
+
+    // If true, enable detailed logging for all communications with the
+    // kubernetes cluster for this account. Defaults to false.
```
Comment (maggieneterval) [inline api/proto/cloudprovider/kubernetes.proto]:
```suggestion
    // Kubernetes cluster for this account. Defaults to false.
```

Comment (maggieneterval) [reply api/proto/cloudprovider/kubernetes.proto]: (just because we are capitalizing it everywhere else)

**Review by maggieneterval — APPROVED** [review]
🎁 🐍 🔢 ❓ ✅

**Pull request merged**